README

```diff
@@ -26,6 +26,8 @@
 3. PREREQUISITES:
 
     Python3 is installed on the System.
+    Note :  If python bin is not present in the system , please create of soft link pointing to python3
```

Comment:
Suggestion: If python binary ... pointing to python<version> binary

Reply:
done, thanks for the review